### PR TITLE
feat(materials): add knobs to material components

### PR DESCRIPTION
**Pull request by zwaardje**
Add knobs to material components
Create knob files per component allow those knobs to be included in the large page templates

### CODE REVIEW

File: src/stories/organisms/colors/colors.knobs.stories.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/* eslint camelcase: ["error", {properties: "never"}] */
+import {text, number, object} from '@storybook/addon-knobs';
+import data from './colors.stories.json';
+
+export default () => ({
+    title: text('Colors - Main title', data.title),
+    material: {
+        title: text('Colors - title', data.material.title),
```
Comment (rwesterhoff) [inline src/stories/organisms/colors/colors.knobs.stories.js]: we don't need this for storybook

File: src/stories/organisms/colors/colors.knobs.stories.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        short_title: text('Colors - short title', data.material.short_title),
```
Comment (rwesterhoff) [inline src/stories/organisms/colors/colors.knobs.stories.js]: we don't need this for storybook

File: src/stories/organisms/colors/colors.knobs.stories.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        weight: {
```
Comment (rwesterhoff) [inline src/stories/organisms/colors/colors.knobs.stories.js]: should be managable for each color

File: src/stories/organisms/colors/colors.knobs.stories.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            value: number('Colors - weight value', data.material.weight.value),
+            unit: number('Colors - weight unit', data.material.weight.unit)
+        },
+        price: {
```
Comment (rwesterhoff) [inline src/stories/organisms/colors/colors.knobs.stories.js]: we don't need this for storybook

File: src/stories/organisms/colors/colors.knobs.stories.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+            value: number('Colors - price value', data.material.price.value),
+            currency: text('Colors - price unit', data.material.price.currency)
+        },
+        _links: {
```
Comment (rwesterhoff) [inline src/stories/organisms/colors/colors.knobs.stories.js]: do we need this for storybook?

File: src/stories/organisms/colors/colors.knobs.stories.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+            self: {
+                href: text('Colors - links href', data.material._links.self.href)
+            }
+        },
+        image: {
```
Comment (rwesterhoff) [inline src/stories/organisms/colors/colors.knobs.stories.js]: do we need this for storybook?

File: src/stories/organisms/hero-product/hero-product.knobs.stories.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
+import {text, number, object} from '@storybook/addon-knobs';
+import data from './hero-product.stories.json';
+
+export default () => ({
+    title: text('Hero - title', data.title),
+    description: text('Hero - description', data.description),
+    materials: object('Hero - pricing', data.materials),
```
Comment (rwesterhoff) [inline src/stories/organisms/hero-product/hero-product.knobs.stories.js]: is it possible to add the 'icon' to the 'pricing'?

File: src/stories/organisms/unique-buying-reasons/ubr.knobs.stories.js [hunk 8]
```diff
@@ -0,0 +1,9 @@
+import {text, object} from '@storybook/addon-knobs';
+import data from './ubr.stories.json';
+
+export default () => ({
+    title: text('UBR - title', data.title),
```
Comment (rwesterhoff) [inline src/stories/organisms/unique-buying-reasons/ubr.knobs.stories.js]: title is currently empty, can you fill it with 'Lorem ipsum dolor sit amet'

File: src/stories/organisms/examples/examples.knobs.stories.js [hunk 9]
```diff
@@ -0,0 +1,8 @@
+import {text, object} from '@storybook/addon-knobs';
+import data from './examples.stories.json';
+
+export default () => ({
+    title: text('Examples - title', data.title),
+    horizontalList: object('Examples - horizontal List', data.horizontalList),
+    cards: object('Examples - cards', data.cards)
```
Comment (rwesterhoff) [inline src/stories/organisms/examples/examples.knobs.stories.js]: In the 'examples' organism the knobs are fin, can you add knobs to the 'card' molecule as well? Containing an image, title, description and link.

File: src/stories/organisms/examples/examples.stories.js [hunk 10]
```diff
@@ -1,28 +1,18 @@
 import {storiesOf} from '@storybook/vue';
 import {withKnobs, text, object} from '@storybook/addon-knobs';
 import data from './examples.stories.json';
```
Comment (rwesterhoff) [inline src/stories/organisms/examples/examples.stories.js]: is this one still necessary if you import it into the xx.knobs.js?

File: src/stories/organisms/colors/colors.stories.js [hunk 11]
```diff
@@ -1,21 +1,19 @@
 import {storiesOf} from '@storybook/vue';
+import {withKnobs} from '@storybook/addon-knobs';
 import data from './colors.stories.json';
```
Comment (rwesterhoff) [inline src/stories/organisms/colors/colors.stories.js]: is this one still necessary if you import it into the xx.knobs.js?

File: src/stories/organisms/unique-buying-reasons/ubr.stories.js [hunk 12]
```diff
@@ -2,26 +2,19 @@ import {storiesOf} from '@storybook/vue';
 import {withKnobs, text, object} from '@storybook/addon-knobs';
 import Ubr from 'organisms-page/ubr';
 import ubrData from './ubr.stories.json';
```
Comment (rwesterhoff) [inline src/stories/organisms/unique-buying-reasons/ubr.stories.js]: is this one still necessary if you import it into the xx.knobs.js?

**Comment by rwesterhoff** [comment]
@zwaardje I did the changes, the only thing not working is the image. It just doesn't show up. Can you have a look at that?

**Comment by zwaardje** [comment]
@rwesterhoff , changed the image from an array to an object. Card required it to be an object to be displayed.

**Pull request merged**